stable_diffusion/stable_diffusion_tpu_req.py

```diff
@@ -0,0 +1,132 @@
+import argparse
+from concurrent import futures
+import functools
+from io import BytesIO  # pylint:disable=g-importing-member
```

Comment:
Can you remove the `pylint` comment here as well?

Reply:
Done in [f6f3577](https://github.com/ray-project/serve_config_examples/pull/8/commits/f6f35770716a61fe57a84b8c182a664ff72cf633).